metadata_backend/api/middlewares.py

```diff
@@ -43,32 +28,20 @@ async def http_error_handler(req: Request, handler: Callable) -> Response:
         response = await handler(req)
         return response
     except web.HTTPError as error:
-        details = _json_exception(error.status, error, req.url)
-        LOG.error(details)
+        LOG.error(error)
+        problem = _json_problem(error, req.url)
+        LOG.info(problem)
```

Comment:
Should we add some description to the logs, and not only the variables?

Reply:
I've been thinking about it, and it's somehow in the back of my mind, but I didn't come up with a solution.